FLEDGE_Key_Value_Server_API.md

```diff
@@ -65,6 +65,12 @@ HTTPS is used to transport data. The method is `POST`.
 
 The HTTP POST body is encrypted.
 
+#### Cleartext headers
```

Comment:
I think "Cleartext headers" is a term that may confuse folks as the browser requires trusted bidding signals fetches to be HTTPS which is encrypted, so these headers are not cleartext over the network.  I'd recommend renaming this section to "HTTP Content Types", although that may be confusing with the next section talking about "Obvlivious HTTP".  I think we need to make it clear this section, and the one above it talking about HTTP POST, are both talking about the outter HTTP layer, and the OHTTP sections are talking about the inner HTTP body.